coala_quickstart/info_extraction/Information.py

```diff
@@ -0,0 +1,18 @@
+from coala_quickstart.info_extraction.Info import Info
+
+
+class VersionInfo(Info):
```

Comment:
So this will kinda conflict with the other PRs right? If you want stuff merged in a particular order, lemme know.